### ISSUE TITLE
Setup the Campaign's change Email_List dropdown

### ISSUE THREAD

**Issue by vaskaloidis** [vaskaloidis/mail-funnel-client#30]
The dropdown has already been created for this - but it needs to be finished. 

- In the right hand side of the Funnel builder there is a dropdown containing the email_lists.
- This needs to be labeled correctly, so make it labeled with an explanation of what it does: This dropdown is set to the email_list for that campaign. If you change the value of that dropdown and click a submit button, the campaign's email_list needs to be updated
- Create the functionality for this feature
- You need to create the method in the `funnel_builder` controller to set an email_list for that campaign
- Add a submit button to the dropdown, so when you select an email_list (other than the pre-selected one, which should be the email_list currently assigned to that campaign, when you click the submit button, it will set the email_list - through an ajax call - to whatever email_list the user selects
- Add the Method to the funnel_builder controller to set a new email_list, connect it with the form mentioned above

**So to summarize** - Finish configuring the ability to display the Campaign's currently selected `Email_List`. It needs to display the currently selected email list, and changing this email_list to a different one and clicking a submit button, needs to change the Campaign's email_list, using an AJAX call.
